osu.Game/Configuration/ScoreMeterType.cs

```diff
@@ -18,5 +18,14 @@ public enum ScoreMeterType
 
         [Description("Hit Error (both)")]
         HitErrorBoth,
+
+        [Description("Colour (left)")]
+        ColourLeft,
+
+        [Description("Colour (right)")]
+        ColourRight,
+
+        [Description("Colour (both)")]
+        ColourBoth,
```

Comment:
Maybe split into "left score meter type" and "right score meter type"?

This would make it easy for one to have hit error on left and color on right.

Reply:
The plan is to allow both on each side too. Not worth changing this enum here yet.

Reply:
Suggesting having two dropdowns: one for type and another one is for position. 
But still out of context of this pr